core-bundle/src/EventListener/RobotsTxtListener.php

```diff
@@ -60,6 +63,11 @@ public function __invoke(RobotsTxtEvent $event): void
             $directiveList = $record->getDirectiveList();
             $directiveList->add(new Directive('Disallow', $this->routePrefix.'/'));
             $directiveList->add(new Directive('Disallow', '/_contao/'));
+
+            if ($this->webDebugToolbarListener && $this->webDebugToolbarListener->isEnabled()) {
+                $directiveList->add(new Directive('Disallow', '/_profiler/'));
+                $directiveList->add(new Directive('Disallow', '/_wdt/'));
+            }
```

Comment:
```suggestion
            $directiveList->add(new Directive('Disallow', '/_profiler/'));
            $directiveList->add(new Directive('Disallow', '/_wdt/'));
```

Do we really need to check whether the toolbar is enabled? I doubt that anyone uses these routes for their own app in production, so we might as well always disallow them.

Reply:
Which would make them public in all the `robots.txt` responses of all Contao setups. I too think that it's better like it is now to only add those in debug mode. Feels cleaner.

Reply:
Okay, fine with me.